wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/class-plugin.php

```diff
@@ -349,7 +344,7 @@ private function save_translation_suggestion_source( $translation, $suggested_tr
 			$suggestion_used .= '_modified';
 		}
 		if ( $suggestion_used ) {
-			gp_update_meta( $translation->id, 'suggestion-used', $suggestion_used, 'translation' );
+			gp_update_meta( $translation->id, 'suggestionused', $suggestion_used, 'translation' );
```

Comment:
Why that change? I think `suggestion-used` is fine. Or maybe just `suggestion`.

Reply:
We resolved to using `_` because the function `gp_sanitize_meta_key` removes `-`.

Reply:
Ah I didn't know!